src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,60 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private final Object[] keyArray = new Object[ARRAY_SIZE];
+    private final Object[] valuesArray = new Object[ARRAY_SIZE];
+    private int arraySize = 0;
 
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < arraySize; i++) {
+            if (key != null && key.equals(keyArray[i]) || key == keyArray[i]) {
+                valuesArray[i] = value;
+                return;
+            }
+        }
+        keyArray[arraySize] = key;
+        valuesArray[arraySize] = value;
+        arraySize++;
     }
 
     @Override
     public V get(K key) {
-        return null;
+        V value = null;
+        boolean check;
+        for (int i = 0; i < ARRAY_SIZE; i++) {
+            if (key == null && valuesArray[i] != null) {
+                check = true;
+            } else {
```

Comment:
```suggestion
        for (int i = 0; i < arraySize; i++) {
            if (key == keys[i] || key != null && key.equals(valuesArray[i]) {
                return your value 
            } 
```
this check is enough)